### PR TITLE
Simplify container image build

### PR DESCRIPTION
**Pull request by ericcurtin**
Simplifying

## Summary by Sourcery

Build:
- Simplify the container image build script by consolidating package installation lists and removing redundant entries.

### CODE REVIEW

File: container-images/scripts/build_llama_and_whisper.sh [hunk 1]
```diff
@@ -1,33 +1,31 @@
 #!/bin/bash
```
[the remark this answers is not in the document]
Comment (rhatdan) [reply container-images/scripts/build_llama_and_whisper.sh:1]: I like the idea of adding set -e to the script.

**Comment by rhatdan** [comment]
LGTM

File: container-images/scripts/build_llama_and_whisper.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (ericcurtin) [reply container-images/scripts/build_llama_and_whisper.sh:1]: It's already there, it's just the AI bot isn't smart enough to detect it:

```
main() {
  set -e
```

[the remark this answers is not in the document]
Comment (rhatdan) [reply container-images/scripts/build_llama_and_whisper.sh:1]: It's Artificial Intelligence, but this does not mean it is smart.

**Pull request merged**